modules/core/src/lib/layer-manager.js

```diff
@@ -344,6 +344,10 @@ export default class LayerManager {
     });
   }
 
+  getActivateViewport() {
```

Comment:
Choose one: `getActiveViewport()` or `activateViewport()`.

Same for instance variable.